### ISSUE TITLE
SelectedDate and StartDate not working

### ISSUE THREAD

**Issue by paulinabukowska** [rebeccaXam/XamForms.Controls.Calendar#15]
Hi! I'm trying to deal with a problem with databinding using this calendar control.

this is how my xaml code looks:
```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:controls="clr-namespace:XamForms.Controls;assembly=XamForms.Controls.Calendar"
             x:Class="PGapp.View.Wydarzenia.WydarzeniaKalendarz">
  <controls:Calendar SelectedDate="{Binding Datee}" DateCommand="{Binding DateChosen}" Padding="10,0,10,0" StartDay="Monday" SelectedBorderWidth="4" DisabledBorderColor="Black" BackgroundColor="White"/>
</ContentPage>
```
and viewmodel:
```
class WydarzeniaKalendarzVM : INotifyPropertyChanged
    {
        public event PropertyChangedEventHandler PropertyChanged;
        public void NotifyPropertyChanged(string propertyName)
        {
            if (PropertyChanged != null)
                PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
        }

        private DateTime _Datee;
        public DateTime Datee
        {
            get
            {
                return _Datee;
            }
            set
            {
                _Datee = value;
                NotifyPropertyChanged("Datee");
            }
        }
        public ICommand DateChosen { get; set; }

        
        public WydarzeniaKalendarzVM(INavigation navigation)
        {
            DateChosen = new Command(DateChosenCmd);
        }

        public void DateChosenCmd()
        {
            DateTime aaa = Datee;
        }
    }
```
the problem is that SelectedDate is always returning Date 01.01.0001. Am I doing something wrong?

(of course the binding is set in WydarzeniaKalendarz.xaml.cs file:
`this.BindingContext = new WydarzeniaKalendarzVM(this.Navigation);`

**Comment by rebeccaXam** [comment]
Hallo paulinabukowska,
```SelectedDate``` is currently a OneWay binding, so it will not trigger the set in your ```Datee``` when a new date is selected. I will change that to TwoWayBinding in 1.0.6.

For now you could use:
```
private DateTime? __Datee;
public DateTime? Datee
{
	get
	{
		return _Datee;
	}
	set
	{
		_Datee = value;
		NotifyPropertyChanged(nameof(Datee));
	}
}

public ICommand DateChosen
{
	get
	{
		return new Command((obj) =>
		{
			Datee = obj as DateTime?;
		});
	}
}
```
_Tip: Use  `NotifyPropertyChanged(nameof(Datee));` instead of `NotifyPropertyChanged("Datee");` it is a new C# 6 feature and is compiler and refactor prove._

**Comment by rebeccaXam** [comment]
Should be fix now in version 1.0.6
